src/styles/header.scss

```diff
@@ -0,0 +1,84 @@
+@import './variables';
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  max-width: 100%;
```

Comment:
The `max-width: 100%;` is redundant because `width: 100%;` is already defined. The `width` property already ensures that the `.header` will not exceed the parent's width.